src/commands/distribute/groups/create.ts

```diff
@@ -0,0 +1,133 @@
+import { AppCommand, CommandResult, ErrorCodes, failure, help, success, shortName, longName, required, hasArg, isCommandFailedResult} from "../../../util/commandline";
+import { MobileCenterClient, models, clientRequest } from "../../../util/apis";
+import { out } from "../../../util/interaction";
+import { inspect } from "util";
+import * as _ from "lodash";
+import * as Pfs from "../../../util/misc/promisfied-fs";
+import { DefaultApp } from "../../../util/profile";
+
+const debug = require("debug")("mobile-center-cli:commands:distribute:groups:create");
+
+@help("Create new distribution group")
+export default class CreateDistributionGroupCommand extends AppCommand {
+  @help("Distribution group name")
+  @shortName("n")
+  @longName("name")
+  @required
+  @hasArg
+  public distributionGroup: string;
+
+  @help("List of testers (space-separated list of e-mails)")
+  @shortName("t")
+  @longName("testers")
+  @hasArg
+  public testers: string;
+
+  @help("Path to file containing list of testers")
+  @shortName("T")
+  @longName("testers-file")
+  @hasArg
+  public testersListFile: string;
+
+  public async run(client: MobileCenterClient): Promise<CommandResult> {
+    try {
```

Comment:
Yes, I think, it should be pulled up. I have created the pull request with this change - https://github.com/Microsoft/mobile-center-cli/pull/158